admin/controller/extension/payment/razorpay.php

```diff
@@ -168,6 +180,107 @@ public function index()
         $this->response->setOutput($this->load->view('extension/payment/razorpay', $data));
     }
 
+    public function autoEnableWebhook()
+    {
+        $webhookExist = false;
+        $webhookUrl   = HTTPS_CATALOG . 'index.php?route=extension/payment/razorpay/webhook';
+
+        $eventsSubscribe = $this->request->post['payment_razorpay_event_selection'];
+
+        $prepareEventsData = [];
+
+        if(empty($eventsSubscribe) == false)
+        {
+            foreach ($eventsSubscribe as $value) 
+            {
+                $prepareEventsData[$value] = true;
+            }
+        }
+
+        if(in_array($_SERVER['SERVER_ADDR'], ["127.0.0.1","::1"]))
```

Comment:
In case of unable to add or update weebhook on RZP dashboard. Make it disable in the configs.